project-fortis-interfaces/src/stores/DataStore.js

```diff
@@ -189,6 +190,7 @@ export const DataStore = Fluxxor.createStore({
     handleReloadChartData(changedData) {
         this.syncChartDataToStore(changedData);
         this.syncFilterSelections(changedData);
+        this.dataStore.sharedLink = changedData.sharedLink;
```

Comment:
Please rename this to something more descriptive, e.g. `dashboardIsLoadedFromShareLink`, so that in the future it's easier to figure out what this is doing.